issue-003/solution.js

```diff
@@ -1,3 +1,9 @@
 function product(a, b) {
     // implement your solution here
+    if (!isNaN(a) && !isNaN(b)) {
```

Comment:
`isNaN` is not the best way to check for something that _is not a number_. It should only be used to check for `NaN` types which are actually _special numbers_. So, you should have used `typeof a === 'number'` here instead.